collection-de-l-ecole-francaise-de-rome-full-note.csl

```diff
@@ -0,0 +1,319 @@
+      <text variable="publisher-place"/>
+    </macro>
+    <macro name="year-date">
+      <choose>
+        <if variable="issued">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="edition">
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="thesis">
+      <group prefix=" " delimiter=", ">
+        <text macro="title" font-style="italic"/>
+        <text variable="genre"/>
+        <text variable="publisher"/>
+        <text macro="year-date"/>
+      </group>
+    </macro>
+    <macro name="articles">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <text variable="volume"/>
+        <text variable="issue"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <group prefix="(" suffix=")" delimiter=", ">
+            <text variable="URL"/>
+            <text macro="access"/>
+          </group>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="book">
+      <group delimiter=", ">
+        <text macro="title"/>
+        <text macro="edition"/>
+        <text macro="publisher"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+      </group>
+    </macro>
+    <macro name="dictionary">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="volume" form="short"/>
+          <text variable="volume"/>
+        </group>
+        <text macro="year-date"/>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="chapter">
+      <group delimiter=" ">
+        <text macro="title" quotes="true" suffix=","/>
+        <text term="in"/>
+        <text macro="editor" suffix=","/>
+        <text variable="container-title" font-style="italic" suffix=","/>
+        <text macro="publisher" suffix=","/>
+        <group delimiter=" " suffix=",">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <citation disambiguate-add-year-suffix="true" et-al-subsequent-min="2" et-al-subsequent-use-first="1">
```

Comment:
> It's specifically the 2 that I'm wondering about, because that's also gramatically wrong for et al. (which stands for and others)\

Is it? Singular for "alia" is "alius", so that would still abbreviate the same. :P (https://en.wiktionary.org/wiki/alius)